packages/nextjs/components/scaffold-eth/RainbowKitCustomConnectButton/AddressInfoDropdown.tsx

```diff
@@ -116,7 +117,10 @@ export const AddressInfoDropdown = ({
             <button
               className="menu-item text-error btn-sm !rounded-xl flex gap-3 py-3"
               type="button"
-              onClick={() => disconnect()}
+              onClick={() => {
+                disconnect();
+                signOut();
```

Comment:
Same problem as https://github.com/BuidlGuidl/ens-pg/pull/55 . Additional info in tg

Reply:
Fixed at https://github.com/BuidlGuidl/SpeedRunEthereum-v2/pull/159/commits/70dbec23c4c835b8b618e3a83c32d24d1125ef19 

Thanks 🙌